### ISSUE TITLE
GO annotation filtering has gone awry

### ISSUE THREAD

**Issue by mah11** [pombase/pombase-chado#557]
The GAF from the latest chado build has vastly too many IEA and NAS annotations, as though the usual filtering isn't happening.

v30 GAF:
1116 NAS
4312 IEA

pombase-build-2016-05-10-v1-l1.gaf:
4947 NAS
14692 IEA

I haven't done counts for other evidence codes, but it's clear something's up (or has gone down. or away).

**Comment by kimrutherford** [comment]
Thanks for spotting that.  The number of lines in the GAF went from 41000 to 33000 on July 14th and then back up to 56000 on July 17th.  I'll look into it.

**Comment by kimrutherford** [comment]
You're right, the filtering isn't working (at all I think).

It's a bit odd.  If I run the GO filtering manually it works fine but it doesn't work from the Chado load script.  I'm trying to work it out.

**Comment by kimrutherford** [comment]
I think I've fixed this.  The filtering was using a table in Chado the wasn't populated until after the go-filtering was done.

It was broken by change pombase/pombase-legacy@b85a9bf6d6d33 where I trying to speed up the stats page.

I've also added a chado-check that will complain if there are more than 40000 GO annotations at the end of the load script, which would mean the filtering has failed.  40000 is a bit arbitrary, we can increase it as needed.

A new load is running now.
